lib/Sema/TypeCheckDecl.cpp

```diff
@@ -2451,16 +2503,40 @@ EmittedMembersRequest::evaluate(Evaluator &evaluator,
   forceConformance(Context.getProtocol(KnownProtocolKind::Decodable));
   forceConformance(Context.getProtocol(KnownProtocolKind::Encodable));
   forceConformance(Context.getProtocol(KnownProtocolKind::Hashable));
+  forceConformance(Context.getProtocol(KnownProtocolKind::Differentiable));
```

Comment:
@dan-zheng @rxwei I think this should be uncontroversial. We need to force the derived members, if any, to be synthesized here, because the result of this request is now cached.

Reply:
Looks good to me.